lib/features/steps/breadcrumb_steps.rb

```diff
@@ -1,5 +1,30 @@
 # @!group Breadcrumb steps
 
+# Tests whether the first event entry contains the specified number of breadcrumbs.
+#
+# @step_input expected [Integer] The expected number of breadcrumbs
+Then("the event has {int} breadcrumb(s)") do |expected|
+  breadcrumbs = Maze::Server.errors.current[:body]['events'].first['breadcrumbs']
+
+  Maze.check.equal(
+    expected,
+    breadcrumbs.length,
```

Comment:
If some notifiers may omit breadcrumbs entirely, then this would error in an unfriendly way.

Reply:
Fixed in a1abcd7